src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 800px;
+  padding: 10px;
+
+  @extend %start-day-sat;
+
+  @for $i from 1 through $month-length {
```

Comment:
Make sure that the variable `$month-length` is defined and holds the correct number of days for the month you are styling. Otherwise, this loop will not work as intended.